views/Data.js

```diff
@@ -32,16 +32,22 @@ let reducer = produce((draft, action) => {
   switch (action.type) {
     case SET: {
       set(draft.value, action.path, action.value)
+      if (action.touched !== false) {
```

Comment:
should we just say `if (action.touched)` and not even pass `touched: false` when it isn't needed?

Reply:
I think I will just remove it in that case, it will only add `touched` if set, otherwise will ignore it